### PR TITLE
Add indices size to the mongo plugin

### PR DESCRIPTION
**Pull request by kris-lab**
- [x] Add an "indexSize" metric
- [x] Add additionally an "indexSizePercentageOfMemory" metric

### CODE REVIEW

**Comment by kris-lab** [comment]
@ppp0 skeleton of mongo total index size and system total memory size. wdyt?

**Comment by kris-lab** [comment]
retest this pleaase

**Comment by kris-lab** [comment]
retest this please

File: lib/bipbip/plugin/mongodb.rb [hunk 1]
```diff
@@ -103,6 +108,23 @@ def slow_query_last_check
...
     end
 
+    # @return [Integer]
+    def total_index_size
+      database_names_ignore = %w(admin system local)
+      database_list = (mongodb_client.database_names - database_names_ignore).map { |name| mongodb_database(name) }
+
+      database_list.reduce(0) do |memo, database|
+        results = database.command('dbstats' => 1)
+        memo += results.documents.first['indexSize']
```
Comment (ppp0) [inline lib/bipbip/plugin/mongodb.rb]: Is += needed? I think reduce is managing memo as an accumulator

**Comment by kris-lab** [comment]
Agree, I have replaced `reduce` with `map-reduce` what gets rid of `memo`.

**Comment by kris-lab** [comment]
@ppp0 please review

**Comment by ppp0** [comment]
CI passing, code looks good

**Comment by njam** [comment]
👍

**Pull request merged**